spec/api/comment_spec.rb

```diff
@@ -7,38 +7,48 @@
 
   describe "comments" do
     before(:each) { init_without_subscriptions }
+
     describe "GET /api/v1/comments/:comment_id" do
       it "returns JSON" do
-        comment = Comment.first
-        get "/api/v1/comments/#{comment.id}"
-        last_response.should be_ok
-        last_response.content_type.should == "application/json;charset=utf-8"
+        Comment.each do |comment|
+          get "/api/v1/comments/#{comment.id}"
+          last_response.should be_ok
+          last_response.content_type.should == "application/json;charset=utf-8"
+        end
       end
       it "retrieve information of a single comment" do
-        comment = Comment.first
-        get "/api/v1/comments/#{comment.id}"
-        last_response.should be_ok
-        retrieved = parse last_response.body
-        retrieved["body"].should == comment.body
-        retrieved["endorsed"].should == comment.endorsed
-        retrieved["id"].should == comment.id.to_s
-        retrieved["children"].should be_nil
-        retrieved["votes"]["point"].should == comment.votes_point
-        retrieved["depth"].should == comment.depth
-        retrieved["parent_id"].should == comment.parent_ids[-1]
+        Comment.each do |comment|
```

Comment:
@macdiesel The reason was that in mongoid5 `Comment.first` doesn't implicitly sort by `_id`, so after switching to mongoid5 this spec was fetching one of the comments that has a parent (the first one doesn't),  and it turned out this line:

    retrieved["parent_id"].should == comment.parent_ids[-1]

wasn't working because in mongoid5 `comment.parent_ids` returns an array of `BSON::ObjectId` objects, not an array of strings. I added the `Comment.each` loop just to make sure the modified code worked correctly for all comments, but there's no point in leaving that in; I'll remove it.